Makefile

```diff
@@ -191,6 +191,8 @@ fmt: format
 lint:
 	if [ $$(wc -l < tests/utils.go) -gt 3565 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
```

Comment:
I think it's better to have that `if` statement separated from  `golangci-lint`. IMHO, `make lint` should run either custom checks or golang linter.

We can have another command that says `golint` (e.g. `make golint`).

Reply:
Both are linters, the target is per function not per tool.

Reply:
Okay, sounds good.